contentcuration/contentcuration/serializers.py

```diff
@@ -357,10 +358,16 @@ class SimplifiedContentNodeSerializer(BulkSerializerMixin, serializers.ModelSeri
         resource_count=ResourceCount(),
         coach_count=CoachCount(),
     )
+    ancestor_query = Metadata(
```

Comment:
I assumed you were going to add the annotation to the existing metadata query. If you would like to go this route, we should also update `to_representation` as that updates the query class when the list serializer is used ([link](https://github.com/learningequality/studio/pull/1728/files#diff-6b2aac70b4dc5863ce6ec91278cba910R267-R278)). We could potentially generalize the handling too.

Reply:
Nice catch- updated!